webhooks/validating/configuration/src/main.rs

```diff
@@ -289,7 +297,7 @@ mod tests {
                 "spec": {
                     "discoveryHandler": {
                         "name": "debugEcho",
-                        "discoveryDetails": "{\"descriptions\": [\"foo\",\"bar\"]}"
+                        "discoveryDetails": "descriptions:\n- \"foo0\"\n- \"foo1\"\n"
```

Comment:
This change is to ensure that validation succeeds regardless of the format of the `discoveryDetails` string, whether serialized json or yaml